x/fees/types/params_test.go

```diff
@@ -0,0 +1,84 @@
+package types
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/stretchr/testify/suite"
+)
+
+type ParamsTestSuite struct {
+	suite.Suite
+}
```

Comment:
nit, no need for a testing `suite` here since you are not sharing fields across the different tests 👍